src/upload.ts

```diff
@@ -40,8 +44,9 @@ const PORTAL_DIRECTORY_FILE_FIELD_NAME = "files[]";
  *
  * @property [endpointUpload] - The relative URL path of the portal endpoint to contact.
  * @property [endpointLargeUpload] - The relative URL path of the portal endpoint to contact for large uploads.
+ * @property [chunkSizeMultiplier=1] - The multiplier for the chunk size. Increase this to upload larger chunks. Note that all valid chunks must be multiplies of the minimum chunk size, so this is a multiplier and not the actual chunk size.
```

Comment:
Nice docstring. I like how it gives complete information and informs the user of potential false assumptions. 👍